src/send_packets.c

```diff
@@ -93,11 +94,37 @@ float2timer (float time, struct timeval *tvp)
     tvp->tv_usec = n * 100000;
 }
 
-/* buffer should be "file_name" */
-int
-parse_play_args (char *buffer, pcap_pkts *pkts)
+static char* find_file(const char* filename)
 {
-    pkts->file = strdup (buffer);
+    if (filename[0] == '/' || !scenario_path) {
+        return strdup(filename);
+    }
+
+    char* fullpath = NULL;
+    if (asprintf(&fullpath, "%s/%s", scenario_path, filename) < 0) {
+        ERROR_NO("Failed to allocate filepath");
+    }
+
+    if (access(fullpath, R_OK) < 0) {
+        free(fullpath);
+        WARNING("SIPp now prefers looking for pcap files next to the scenario. "
+                "{} couldn't be found next to the secnario, falling back to "
```

Comment:
Since when does vprintf take python-format-style `{}` parameters? (Shouldn't this have thrown a `-Wformat-extra-args` error?)

And "secnario".